docs/rst/tutorials/cloud/router_max_tx_rate/router_max_tx_rate.rst

```diff
@@ -2,8 +2,15 @@
 
 .. _tutorials_cloud_manual_topics:
 
-ROS 2 Router configuration with Manual Topics
-=============================================
+.. NOT-YET-AVAILABLE-TUTORIAL
+
+.. warning::
+
+    The |rosrouter| features described in this tutorial are not yet available in the current release of Vulcanexus.
+    Stay tuned for new Vulcanexus releases to start using these new features!
+
+Configuring DDS Topics in the ROS 2 Router
```

Comment:
I think the title of the tutorial is too broad. Maybe `Configuring DDS Topic QoS in the ROS 2 Router` would be more descriptive?

Reply:
Update to "Configuring ROS 2 Router's Topic QoS"